app/views/component/locations/detail/_body.html.haml

```diff
@@ -280,13 +280,15 @@
 
   initPrintButton();
 
-  
+
   document.querySelectorAll('.service-row').forEach( row => {
     row.addEventListener("click", toggleServiceCard);
+    row.addEventListener("keydown", toggleServiceCard);
```

Comment:
this automatically open when we tab on then given that any keydown will toggle it to be expanded, I winder if we should change this to be only on enter?

I found a very good [article/book](https://inclusive-components.design/collapsible-sections/) about the expandable section so the screen reader will identify that it is collapsed